### PR TITLE
Update statistics.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated to add playbook statistics.

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Thanks, @justinegeffen!

File: source/manage/statistics.rst [hunk 1]
```diff
@@ -87,6 +87,12 @@ Master DB Conns
 Replica DB Conns
     The number of active connections currently on one or more of `your read replica databases <https://docs.mattermost.com/scale/high-availability-cluster.html#database-configuration>`__.
 
+Playbook count
```
Comment (trilopin) [inline source/manage/statistics.rst:90]: Not sure about this, but I think we used the copies "Total Playbooks" and "Total Playbook Runs". 

If you still want to reference as  xxx count, ignore my comment

Comment (trilopin) [reply source/manage/statistics.rst:90]: screenshot-comm

Comment (justinegeffen) [inline source/manage/statistics.rst:90]:
```suggestion
Total Playbooks
```

File: source/manage/statistics.rst [hunk 2]
```diff
@@ -87,6 +87,12 @@ Master DB Conns
...
+    The total number of playbooks on this server.
+
+Playbook runs count
```
Comment (justinegeffen) [inline source/manage/statistics.rst:93]:
```suggestion
Total Playbook Runs
```

**Pull request merged**